app/models/session.rb

```diff
@@ -122,8 +122,7 @@ class Session < ApplicationRecord
                 location.generic_clinic?
             end
 
-  validates :programmes, presence: true
-  validate :programmes_part_of_organisation
```

Comment:
Do we not still want to validate that the programmes are a part of the organisation?

Reply:
The organisation programmes are current, but I can imagine a scenario where an organisation used to administer a programme but doesn't anymore, and we wouldn't want all the historical sessions to be invalid.